CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.17)
 set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
-
-project(FLCL VERSION 0.3.0 LANGUAGES Fortran C CXX)
+set(FLCL_VERSION 0.4.0)
```

Comment:
You don't need to set a variable, you can use `PROJECT_VERSION`
https://cmake.org/cmake/help/latest/variable/PROJECT_VERSION.html